irksome/ButcherTableaux.py

```diff
@@ -24,10 +24,6 @@ def __init__(self, A, b, btilde, c, order):
         self.btilde = btilde
         self.c = c
         self.order = order
-        try:
-            self.Ainv = numpy.linalg.inv(A)
-        except numpy.linalg.LinAlgError:
-            self.Ainv = None
 
```

Comment:
I think that all of the code that relied on this has been removed in this PR, so we should remove this as well.

Reply:
`grep -iR ainv */*py` suggests that this is the case.  If we need it again in the future it's easy to put back.